x-pack/plugins/upgrade_assistant/public/application/components/es_deprecations/deprecation_types/reindex/use_reindex_state.tsx

```diff
@@ -0,0 +1,187 @@
+/*
```

Comment:
Refactoring of `polling_service.ts`. Work initially started as part of https://github.com/elastic/kibana/pull/99245.